worker/src/Worker.cpp

```diff
@@ -451,6 +451,28 @@ inline void Worker::OnSignal(SignalsHandler* /*signalsHandler*/, int signum)
 
 			Close();
 
+			// Now cleanup the sockets that are created for this process.
+			std::string dir = "" ;
+			if(!std::getenv("MEDIASOUP_SOCKET_DIR"))
+			{
+				dir = "tmp";
+			}
+			else
+			{
+				dir = std::getenv("MEDIASOUP_SOCKET_DIR");
+
+			}
+
+			std::string socketName = GetUnixSocketName();
+			std::string delSocket_string = "/";
+			delSocket_string.append(dir);
+			delSocket_string.append("/");
+			delSocket_string.append(socketName);
+
+			if (unlink(delSocket_string.c_str()) != 0)
+			      fprintf(stderr, "unlink() error");
```

Comment:
you can use strerror to print the errno description in this case
can we use the log macro instead of printf or isn't too late here ?

Reply:
Will use the MS_ERROR_STD. Forgot to replace it after debugging.